pinc/ProofreadingToolbox.inc

```diff
@@ -4,31 +4,69 @@ include_once($relPath.'faq.inc');
 include_once($relPath.'stages.inc');
```

Comment:
Is this still needed? I suspect this was for `is_formatting_round()`, but I'm not 100% certain.

Reply:
I've removed it. Still works.